package.json

```diff
@@ -37,9 +37,10 @@
   },
   "homepage": "https://github.com/nkbt/react-page-click",
   "peerDependencies": {
-    "react": "^0.14 || ^15"
+    "react": ">=15.3"
   },
   "devDependencies": {
+    "prop-types": "^15.5.8",
```

Comment:
Wanted to merge, but just noticed that `prop-types` is in `devDependencies`. It needs to be in `dependencies`

Sorry for missing it right away